app/constants.py

```diff
@@ -0,0 +1,182 @@
+                                IDs."
+
+# comment = update
+NO_DATA_FOR_UPDATING_PROFILE_WAS_SENT = "No data for updating profile was sent"
+
+# comment = Relation constraints'
+MENTOR_ID_SAME_AS_MENTEE_ID = "You cannot have a mentorship relation with \
+                                yourself."
+END_TIME_BEFORE_PRESENT = "End date is invalid since date has passed."
+MENTOR_TIME_GREATER_THAN_MAX_TIME = "Mentorship relation maximum duration is 6 \
+                                     months."
+MENTOR_TIME_LESS_THAN_MIN_TIME = "Mentorship relation minimum duration is 4 \
+                                  week."
+CANT_ACCEPT_MENTOR_REQ_SENT_BY_HIMSELF = "You cannot accept a mentorship \
+                                          request sent by yourself."
+CANT_ACCEPT_UNINVOLVED_MENTOR_RELATION = "You cannot accept a mentorship \
+                                          relation where you are not involved."
+USER_CANT_REJECT_REQUEST_FOR_MENTOR = "You cannot reject a mentorship request \
+                                       sent by yourself."
+CANT_REJECT_UNINVOLVED_RELATION_REQUEST = "You cannot reject a mentorship \
+                                           relation where you are not \
+                                           involved."
+CANT_CANCEL_UNINVOLVED_REQUEST = "You cannot cancel a mentorship relation \
+                                  where you are not involved."
+CANT_DELETE_UNINVOLVED_REQUEST = "You cannot delete a mentorship request that \
+                                  you did not create."
+NOT_IN_MENTORED_RELATION_CURRENTLY = "You are not in a current mentorship \
+                                      relation."
+USER_IS_INVOLVED_IN_A_MENTORSHIP_RELATION = "You are currently involved in a \
+                                            mentorship relation."
+USER_NOT_INVOLVED_IN_THIS_MENTOR_RELATION = "You are not involved in this \
+                                             mentorship relation."
+USER_USES_A_USERNAME_THAT_ALREADY_EXISTS = "A user with that username already \
+                                            exists."
+USER_USES_AN_EMAIL_ID_THAT_ALREADY_EXISTS = "A user with that email already \
+                                            exists."
+USER_IS_NOT_REGISTERED_IN_THE_SYSTEM = "You are not registered in the system."
+NAME_LENGTH_GREATER_THAN_MAX_LIMIT = "The {field_name} field has to be shorter \
+                                      than {max_limit} characters."
+NAME_LENGTH_LESSER_THAN_MAX_LIMIT = "The {field_name} field has to be longer \
+                                     than {min_limit} characters and \
+                                     shorter than {max_limit} characters."
+USER_INPUTS_INCORRECT_CONFIGURATION_VALUE = "The environment config value has \
+                                            to be within these values: prod, \
+                                            dev, test."
+
+# comment = Mentorship state."
```

Comment:
why do you have `comment` keyword in the comment here and in other places of this file?

Reply:
I'll resolve this @isabelcosta :+1: